src/mdlpp/structs/VTX.cpp

```diff
@@ -124,20 +149,37 @@ bool VTX::open(const std::byte* data, std::size_t size, const MDL::MDL& mdl) {
 
 							const auto indicesCount = stream.read<int32_t>();
 							stream.read(strip.indicesOffset);
-							// todo: check if offset is in bytes
+							// Note: offset is in elements, not bytes
 							strip.indices = std::span(stripGroup.indices.begin() + strip.indicesOffset, indicesCount);
 
 							const auto verticesCount = stream.read<int32_t>();
 							stream.read(strip.verticesOffset);
-							// todo: check if offset is in bytes
+							// Note: offset is in elements, not bytes
 							strip.vertices = std::span(stripGroup.vertices.begin() + strip.verticesOffset, verticesCount);
 
 							stream
 								.read(strip.boneCount)
 								.read(strip.flags);
 
-							// todo: bone stuff
-							stream.skip<int32_t>(2);
+							// do: bone stuff
```

Comment:
Ok it is pretty funny in the context of this PR but the codebase will last longer